setup.cfg

```diff
@@ -75,6 +75,11 @@ apache.hive =
     impyla
 microsoft.azure =
     apache-airflow-providers-microsoft-azure
+openlineage =
+    openlineage-airflow==0.6.2
+# Library to run a flask based local Lineage server to dump events to.
+flask =
+    Flask==1.1.4
```

Comment:
Flask is a requirment of Airflow, and since we only need it for dev, we shouldn't have it separately and not as an extra anyway

Reply:
https://github.com/astronomer/astronomer-providers/pull/325